ZadanieZespolowe1.Rmd

```diff
@@ -22,10 +22,43 @@ bezrobocie_typ <- "b"
 Celem poniższego dokumentu jest analiza danych dotyczących stopy bezrobocia w Polsce na podstawie danych GUS i prognoza wysokości stopy na rok `r rok`.
 
 ## Wczytanie danych
+Dane zostały wczytane i odpowiednio zmodyfikowane w oddzielnym R skrypcie. Tabela została wyodrębniona ze strony z usunięciem niepotrzebnych znaków, ujednoliceniem separatora dziesiętnego i dodaniem możliwości wyboru wariantu w roku 2002:
+
+w wariancie "a"- dane według wyników z Powszechnego Spisu Rolnego 1996,
+
+w wariancie "b"- dane według wyników z Narodowego Spisu Powszechnego Ludności i Mieszkań oraz Powszechnego Spisu Rolnego 2002
+
+W naszych rozważaniach domyślnie weźmiemy pod uwagę wariant "b".
```

Comment:
To jak robimy kod uniwersalny, to fajnie byłoby mieć to "b" sparametryzowane, w razie jakby ktoś chciał jednak zrobić na "a". Można w to miejsce wrzucić inline chunka z `bezrobocie_typ`.